boards/ti/lp_em_cc2340r5/Kconfig.lp_em_cc2340r5

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 Texas Instruments Incorporated
+# Copyright (c) 2024 BayLibre, SAS
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_LP_EM_CC2340R5
+	select SOC_CC2340R5
+
+config BOARD_USE_LF_XOSC
+	bool "Use external low-frequency crystal"
+	default y
```

Comment:
this belongs to DT

Reply:
OK. I will remove it from this PR, since it is used nowhere yet.

We will add it to DT when we need it (that will be the case for a conditional build related to power management, which will be submitted upstream later).